model/map-file/src/main/java/org/keycloak/models/map/storage/file/FileCriteriaBuilder.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2022 Red Hat, Inc. and/or its affiliates
```

Comment:
nitpick, it applies to other new files as well
```suggestion
 * Copyright 2023 Red Hat, Inc. and/or its affiliates
```